### ISSUE TITLE
Support for Python 3.9

### ISSUE THREAD

**Issue by bl-nkd-v** [jetbridge/psycopg2-lambda-layer#43]
Regarding the PR that was merged yesterday, https://github.com/jetbridge/psycopg2-lambda-layer/pull/42 
Seems like the 3.9 version isn't yet available

![Screenshot 2023-08-23 at 11 27 40 AM](https://github.com/jetbridge/psycopg2-lambda-layer/assets/77947741/a8e191e2-8670-4dcb-aa51-87b3e814078c)

(vs 3.8):
![Screenshot 2023-08-23 at 11 28 16 AM](https://github.com/jetbridge/psycopg2-lambda-layer/assets/77947741/aff63233-eada-42d6-9be5-0093062b9f10)

Will the layers for Python 3.9 be redeployed anytime soon? Running into issues with compiling my own psycopg2 lambda layers, and this would be extremely useful to have! The region I'm particularly interested in is `us-west-1`. Appreciate it!

**Comment by jpines-vi** [comment]
I have the same in us-east-2.

**Comment by ToniB-M3ter** [comment]
anything for us-east-1?

**Comment by revmischa** [comment]
Also please see: https://github.com/jetbridge/psycopg2-lambda-layer/issues/31

**Comment by revmischa** [comment]
I haven't tested it but I merged #45 and deployed some layers. LMK if they work for you. Thanks for your patience.

```
      # py 3.9:
      - arn:aws:lambda:us-east-1:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:us-east-2:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:us-west-1:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:us-west-2:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:ca-central-1:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:eu-west-1:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:eu-west-2:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:eu-west-3:898466741470:layer:psycopg2-py39:1
      - arn:aws:lambda:eu-central-1:898466741470:layer:psycopg2-py39:1
```

**Comment by andrelmbraga** [comment]
Did not work for me:
      - arn:aws:lambda:us-east-2:898466741470:layer:psycopg2-py39:1
